safe/definitions/extra_keywords.py

```diff
@@ -375,6 +387,11 @@
     extra_keyword_volcano_longitude,
 ]
 
+flood_extra_keywords = [
+    extra_keyword_flood_event_id,
+    extra_keyword_volcano_eruption_event_time,
```

Comment:
Hi @ismailsunni - why does the volcano eruption event time is added as the flood extra keywords?

Reply:
Ah yes, sorry, forget to add then the autocomplete finish it :D. Fixed in the incoming commit.